dockstore-integration-testing/src/test/java/io/dockstore/webservice/ZenodoIT.java

```diff
@@ -129,6 +131,7 @@ public static void dropAndRecreateDB() throws Exception {
     @BeforeEach
     public void resetDBBetweenTests() throws Exception {
         CommonTestUtilities.cleanStatePrivate2(SUPPORT, false, testingPostgres);
+        SUPPORT.getConfiguration().getGitHubOrgsWithDoiGeneration().clear();
```

Comment:
Neat! When I suggested modifying it this way, I wasn't sure if it was going to carry over to the Dockstore setting between tests

Reply:
That was a good suggestion, that I initially (incorrectly) dismissed. I had thought it was changing the file contents, but it is just changing a Java property.

But yes, there's just one instance of it, and my failing to reset here was causing tests to fail.